src/features/channels/components/chat-content/ChatContent.tsx

```diff
@@ -56,28 +49,24 @@ const ChatContent = () => {
 
   return (
     <Paper onScroll={onScroll} ref={containerRef} className={classes.main} square>
-      {sortedMessages.map(
-        (
-          { senderAlias, message, createdAt, senderAddress, replyMessage, replyRecipientAlias },
-          index
-        ) => (
-          <Message
-            // TODO: the backend is not giving us any way to differentiate between messages... Too bad!
-            // eslint-disable-next-line react/no-array-index-key
-            key={index}
-            senderAddress={senderAddress}
-            name={senderAlias}
-            message={message}
-            date={formatDate(createdAt)}
-            color="#A36300"
-          >
-            {/* TODO: there must be a better way to achieve this */}
-            {replyMessage && replyRecipientAlias && (
-              <MessageReply name={replyRecipientAlias} message={replyMessage} color="#A36300" />
-            )}
-          </Message>
-        )
-      )}
+      {sortedMessages.map((message, index) => (
+        <Message
+          // TODO: the backend is not giving us any way to differentiate between messages... Too bad!
+          // eslint-disable-next-line react/no-array-index-key
```

Comment:
Do not use inline eslint disable commands. It is better to follow the rule, or there is a very good reason to not to do so, create a general rule to disable it

Reply:
There's no messageId when a new message gets added to a channel through the socket... So, for this time I had to disable eslint for that line. This is a design problem from the backend